filebeat/tests/system/test_multiline.py

```diff
@@ -71,6 +71,38 @@ def test_c_style_log(self):
         # Check that output file has the same number of lines as the log file
         assert 4 == len(output)
 
+    def test_nginx_multiline_log(self):
+        """
+        Test an nginx multiline log
+        Special about this log file is that it has empty new lines
+        """
+        self.render_config_template(
+                path=os.path.abspath(self.working_dir) + "/log/*",
+                multiline=True,
+                pattern="^=[A-Z]+",
+                match="after",
+                negate="true",
```

Comment:
About the newlines: I would say it is not the job of filebeat to cleanup log files, means not sending empty lines (keep magic to a minimum).

I much prefer the solution you proposed which makes it possible to handle the problem through configuration and not changes in the logic. We should probably add these examples to our regexp docs (@dede)